### PR TITLE
SEO-181938-WPF-int-changes

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK ,

Worked on wpf changes.

Regards,
Sharly A.

### CODE REVIEW

File: wpf/DataPager/Overview.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+description: Learn here all about overview of Syncfusion WPF DataPager (SfDataPager) control, its elements and more.
 platform: wpf
 control: SfDataPager Control
 documentation: ug
 ---
 
-# WPF DataPager (SfDataPager) Overview
+# Overview of WPF DataPager (SfDataPager)
 
 The SfDataPager control provides a configurable user interface for paging using a data collection. You can bind the SfDataPager to any IEnumerable <a>(http://msdn.microsoft.com/en-us/library/system.collections.ienumerable(v=vs.95).aspx)collection</a>. The SfDataPager control wraps the collection internally in PagedCollectionView and exposes by using PagedSource property. PagedCollectionView helps to provide the paging functionality. You can apply paging for data bound control by setting PagedSource property as ItemsSource for that control.
```
Comment (MallikaRK) [inline wpf/DataPager/Overview.md:12]: Can you recheck the URL?
image

**Review by MallikaRK — CHANGES_REQUESTED** [review]
commented

File: wpf/DataPager/Overview.md (hunk 1 above)
Comment (Sharly-Atieno) [reply wpf/DataPager/Overview.md]: resolved

**Review by MallikaRK — APPROVED** [review]
approved

**Review by Aishwarya-Ganesan — APPROVED** [review]

**Pull request merged**